utils/getOffchainPrice.ts

```diff
@@ -0,0 +1,30 @@
+const fetch = require("node-fetch");
+const { utils } = require("ethers");
+
+// TODO: This is a really simple mapping of identifier to URL to query to get latest price for an identifier.
+// Future work should blend off-chain prices from different sources similar to how we do it in
+// `protocol/financial-templates-lib/price-feed/MedianizerPriceFeed.js`
+interface PriceFeedIdentifierMap {
+  [identifier: string]: string;
+}
+
+// TODO: Currently using the coinbase API instead of CryptoWatch because CryptoWatch has strict CORS
+// policy that prevents requests coming from localhost domains.
```

Comment:
This LGTM.

Thoughts on ultimately pulling in `@umaprotocol/financial-templates-lib` down the road to use our common set of price feeds?

Reply:
Definitely but right now the fact that we have to pass in a Logger object and a web3 object is a bit overkill. I don't know if this repo even has an easy web3 object to access.

Reply:
This is a good point, and one I hadn't considered. It would be ideal to get all of our libraries to grab default instances of both of these if the user chooses not to provide them.